test/hotspot/jtreg/runtime/ErrorHandling/TestHeapDumpOnOutOfMemoryError.java

```diff
@@ -96,11 +97,25 @@ static void test(String type) throws Exception {
         output.stdoutShouldNotBeEmpty();
         output.shouldContain("Dumping heap to " + type + ".hprof");
         File dump = new File(heapdumpFilename);
-        Asserts.assertTrue(dump.exists() && dump.isFile(),
-                "Could not find dump file " + dump.getAbsolutePath());
-
+        Asserts.assertTrue(dump.exists() && dump.isFile(), "Could not find dump file " + dump.getAbsolutePath());
```

Comment:
I. think you could just update the test to use
heapdumpFilename = type + ".%p.hprof";
we don't need test twice, it is quite expensive.

Reply:
sure, it's a few seconds but we can save that time 8-)